compiler/compiler_test.go

```diff
@@ -16,11 +16,11 @@ var CmpOptions = semantictest.CmpOptions
 
 func TestCompileAndEval(t *testing.T) {
 	testCases := []struct {
-		name    string
-		fn      string
-		inType  semantic.MonoType
-		input   values.Object
-		want    values.Value
+		name           string
+		fn             string
+		inType         semantic.MonoType
+		input          values.Object
+		want           values.Value
```

Comment:
change made by go fmt

Reply:
I feel like I've seen this before. I wonder if this is due to inconsistent go versions among the development team.